### PR TITLE
add README .

### PR DESCRIPTION
**Pull request by Tony041010**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+CFOP is a common method to solve a Rubik’s cube used by top cubers. CFOP stands for the 4 steps to solve a cube - Cross, F2L, OLL, and PLL. Solving a cube under 20 seconds with CFOP requires the person to muscle-memorize 78 algorithms, to recognize patterns speedy, etc, which requires practices and good learning strategies. In this project, we identify 7 essential OLL algorithms and build a learning tool based on them.
+# Overview
+
+CubeTrainer v0.1 trains a learner to muscle-memorize the 7 essential OLL algorithms by doing the followings repeatedly:
+
+Selects one algorithm using sorting strategy S,
+
+Shows the selected algorithm to the user, 
+
+Collects the time that the user spends on memorizing the algorithm to update sorting strategy S.
```
Comment (heronyang) [inline README.md:14]: Use `S` instead.

**Review by heronyang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Tony041010) [reply README.md:14]: resolved.

**Review by heronyang — APPROVED** [review]
LGTM.

**Review by heronyang — APPROVED** [review]
Try again.

**Pull request merged**